integreat_cms/cms/templatetags/content_filters.py

```diff
@@ -30,6 +30,11 @@ def get_translation(instance, language_slug):
     return instance.get_translation(language_slug)
 
 
+@register.simple_tag
+def get_poi_translations(instance):
+    return instance.translations
+
+
```

Comment:
```suggestion
```
I don't think we need a custom template tag for a simple reverse foreign key... You can just use `poi.translations` in the template directly.